.github/workflows/optimize-release-optimize-c8-only.yml

```diff
@@ -145,6 +146,21 @@ jobs:
           username: ${{ steps.secrets.outputs.REGISTRY_HUB_DOCKER_COM_USR }}
           password: ${{ steps.secrets.outputs.REGISTRY_HUB_DOCKER_COM_PSW }}
 
+      - id: 'auth'
+        name: 'Authenticate to Google Cloud'
+        uses: 'google-github-actions/auth@v2'
+        with:
+          token_format: 'access_token'
+          workload_identity_provider: 'projects/271764561088/locations/global/workloadIdentityPools/github/providers/camunda'
+          service_account: 'github-actions@team-infosec.iam.gserviceaccount.com'
```

Comment:
```suggestion
          service_account: 'github-actions-camunda@team-infosec.iam.gserviceaccount.com'
```

Reply:
I've created a service account for this workflow to have limited access.

Reply:
hey @jjettenCamunda did you made all the test you needed? Are you okay if I merge this PR?